src/_configure_plextrac.sh

```diff
@@ -28,6 +28,7 @@ function generate_default_config() {
 
   # Generate base env, using imported vars from above where applicable
   generatedEnv="
+API_INTEGRATION_AUTH_CONFIG_NOTIFICATION_SERVICE=${API_INTEGRATION_AUTH_CONFIG_NOTIFICATION_SERVICE:-`generateSecret`}
```

Comment:
Will this be a separate key for each service? I don't know how the app uses it, but I want to be sure this works the way you expect.

Reply:
Yes, a separate key for each service that needs to use one should be generated.